ramls/inventory-event-handlers.raml

```diff
@@ -9,20 +9,6 @@ documentation:
     content: <b>API for handling events</b>
 
 /inventory/handlers:
-  /data-import:
-    displayName: API for handling all data-import events
-    post:
-      description: "Handler for data-import events"
-      body:
-        application/json:
-      responses:
-        204:
-          description: "Received data-import event"
-        500:
-          description: "Internal server error"
-          body:
-            text/plain:
-              example: "Internal server error"
```

Comment:
Is there code to be removed now that this endpoint has gone?

Reply:
Hi @marcjohnson-kint, 

All necessary changes were made in https://github.com/folio-org/mod-inventory/pull/298, this is merely an oversight - we didn't update raml and didn't change API version at the time

Reply:
@KaterynaSenchenko Thanks